apps/api/src/services/data-provider/manual/manual.service.ts

```diff
@@ -272,10 +274,13 @@ export class ManualService implements DataProviderInterface {
           } catch {}
         }
 
-        return extractNumberFromString({
-          locale,
-          value: $(scraperConfiguration.selector).first().text()
-        });
+        return (
+          factor *
+          extractNumberFromString({
+            locale,
+            value: $(scraperConfiguration.selector).first().text()
+          })
+        );
       }
```

Comment:
Can you please refactor that the postprocessing logic is only applied once after `if` / `else`?